ci-operator/step-registry/jboss-web-server/lp-interop-tests/jboss-web-server-lp-interop-tests-commands.sh

```diff
@@ -0,0 +1,123 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+echo "Run JWS Interop testing..."
+SECRETS_DIR="/tmp/secrets/tests"
+mkdir -p  ${ARTIFACT_DIR}/jws_artifacts/jws-5
+export KUBECONFIG=$SHARED_DIR/kubeconfig
+
+export PATH=$PATH:/tmp/bin
+mkdir -p /tmp/bin
+curl -sL https://github.com/stedolan/jq/releases/download/jq-1.6/jq-linux64 > /tmp/bin/jq
+chmod ug+x /tmp/bin/jq
+
+
+CONSOLE_URL=$(cat $SHARED_DIR/console.url)
+OCP_API_URL="https://api.${CONSOLE_URL#"https://console-openshift-console.apps."}:6443"
+OCP_CRED_USR="kubeadmin"
+OCP_CRED_PSW="$(cat ${SHARED_DIR}/kubeadmin-password)"
+
+oc login ${OCP_API_URL} --username=${OCP_CRED_USR} --password=${OCP_CRED_PSW} --insecure-skip-tls-verify=true
+
+OCP_API_TOKEN=$(oc whoami -t)
+
+export OCP_API_TOKEN
+
+username="$(cat ${SECRETS_DIR}/username)"
+password="$(cat ${SECRETS_DIR}/password)"
+
+#namespaces used for running jws tests on test cluster
+namespaces=("jws-test-image" "jws-tests" "jws-tests-build")
```

Comment:
You can create a variable for these in the ref for better maintenance & logic below

Example: https://github.com/openshift/release/blob/master/ci-operator/step-registry/distributed-tracing/install/jaeger-product/distributed-tracing-install-jaeger-product-ref.yaml

Reply:
done